src/storage/client/StorageClient.cpp

```diff
@@ -513,15 +513,15 @@ StorageClient::get(GraphSpaceID space,
                            });
 }
 
-folly::SemiFuture<StorageRpcResponse<storage::cpp2::LookUpVertexIndexResp>>
+folly::SemiFuture<StorageRpcResponse<storage::cpp2::LookUpIndexResp>>
 StorageClient::lookUpVertexIndex(GraphSpaceID space,
                                  IndexID indexId,
```

Comment:
Could we unify `StorageClient::lookUpVertexIndex` and `StorageClient::lookUpEdgeIndex` to `StorageClient::lookupIndex`.
They are have same code now.